### PR TITLE
[build] Bumps mcs bootstrap requirement

### PR DESCRIPTION
**Pull request by marek-safar**

### CODE REVIEW

File: mcs/build/common/basic-profile-check.cs [hunk 1]
```diff
@@ -40,7 +40,14 @@ static int Main ()
 		if (!Version.TryParse (field.GetValue (null) as string, out version))
 			return 4;
 
-		if (version < new Version (4, 9))
+		Version min_mono_version;
+#if __MonoCS__
+		min_mono_version = new Version (4, 9);
+#else
+		min_mono_version = new Version (5, 4);
```
Comment (akoeplinger) [inline mcs/build/common/basic-profile-check.cs]: shouldn't it be the other way around?

Comment (marek-safar) [inline mcs/build/common/basic-profile-check.cs]: Yes!

**Comment by marek-safar** [comment]
@akoeplinger did we disable mcs build config?

**Comment by akoeplinger** [comment]
@marek-safar on the PRs yes, since it was red for 2+months and just distracting. I can reenable it anytime.

**Pull request merged**